CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+# Contributing to this project
+
+Please take a moment to review this document in order to make the contribution
+process easy and effective for everyone involved.
+
+Following these guidelines helps to communicate that you respect the time of
+the developers managing and developing this open source project. In return,
+they should reciprocate that respect in addressing your issue or assessing
+patches and features.
+
+## Development environment
+
+See [Pre-requisites](README.md#pre-requisites) and
+ [Setup Training Toolbox for TensorFlow](README.md#setup-training-toolbox-for-tensorflow) sections.
+
+## Branching model
+
+The project uses [a successful Git branching model](https://nvie.com/posts/a-successful-git-branching-model).
+Thus it has a couple of branches. Some of them are described below:
+
+- `origin/master` to be the main branch where the source code of
+HEAD always reflects a production-ready state.
+- `origin/develop` to be the main branch where the source code of
+HEAD always reflects a state with the latest delivered development
+changes for the next release. Some would call this the “integration branch”.
+
+## Bug reports
+
+A bug is a _demonstrable problem_ that is caused by the code in the repository.
+Good bug reports are extremely helpful - thank you!
+
+Guidelines for bug reports:
+
+1. **Use the GitHub issue search** &mdash; check if the issue has already been
+   reported.
+
+2. **Check if the issue has been fixed** &mdash; try to reproduce it using the
+   latest `develop` branch in the repository.
+
+3. **Isolate the problem** &mdash; ideally create a reduced test case.
+
+A good bug report shouldn't leave others needing to chase you up for more
+information. Please try to be as detailed as possible in your report. What is
+your environment? What steps will reproduce the issue? What browser(s) and OS
+experience the problem? What would you expect to be the outcome? All these
+details will help people to fix any potential bugs.
+
+Example:
+
+> Short and descriptive example bug report title
```

Comment:
I believe, github has issue templates to guide the contribution

Reply:
I don't see any templates.